test.css

```diff
@@ -0,0 +1,127 @@
+body, html {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  background-color: black;
+}
+
+.background-layer {
+  position: absolute;
+  top: -110px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-image: url('./hackerWaterfall.png');
+  opacity: 0.2;
+  z-index: 0; /* Ensure the background layer is behind content */
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  height: 100vh;
+}
+
+.grid-cross-item {
+  background-image: url('./redlines.png');
+  background-size: 200% 200%; /* Adjust this property according to your needs */
+  z-index: 1;
+}
+
+.grid-fill-item {
+  background-color: rgba(255, 0, 0, 0.8);
+  background-image: url('./redlines.png');
+  background-size: 200% 200%; /* Adjust this property according to your needs */
+  z-index: 1;
+}
+
+.bottom-left {
+  position: fixed;
+  bottom: 0;
+  left: 25px;
+  padding: 40px;
+  color: #fff;
+}
+
+.fira-mono-regular {
+  font-family: "Fira Mono", monospace;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 0.2;
+  font-size: 14px;
+  color: #737373;
+  z-index: 1;
+}
+
+.logo {
+  width: 100px; /* Adjust width as needed */
+  height: auto; /* Maintain aspect ratio */
+  fill: white; /* Set fill color to white */
+  z-index: 2;
+  position: absolute;
+    bottom: 50%;
+    right: 50%;
+}
+
+.headingStuff {
+  font-family: "Fira Mono", monospace;
+  color: white;
+  z-index: 2;
+  text-align: right;
+  vertical-align: text-top;
+  font-size: 26px;
+  position: relative;
```

Comment:
Try not to force position with position:relative. Mess around with padding, gap and margin instead. 

The main use case of position:relative is just on parents of position:absolute elements... honestly don't see it used much else.
[https://css-tricks.com/absolute-positioning-inside-relative-positioning/](https://css-tricks.com/absolute-positioning-inside-relative-positioning/)